conda_forge_tick/new_update_versions.py

```diff
@@ -78,29 +81,24 @@ def new_update_upstream_versions(
     for node, node_attrs in tqdm.tqdm(_all_nodes):
         with node_attrs["payload"] as attrs:
             if attrs.get("bad") or attrs.get("archived"):
-                # "new_version" = False
                 continue
             to_update.append((node, attrs))
 
     for node, node_attrs in to_update:
         # checking each node
         with node_attrs as attrs:
-            up_to = {}
+            version_data = {}
 
             # avoid
             if node == "ca-policy-lcg":
```

Comment:
you might consider putting this clause in the `get_latest_version` function.

Reply:
the `ca-policy-lcg` ?

Reply:
Yes, that carve out is not part of the for loop but a special case that should be in the `get_latest_version` function

Reply:
Oh, now I see. I will commit it later on the pool PR updates